NHHall/nh_hall.hpp

```diff
@@ -566,7 +566,6 @@ class VariableAllpass : public BaseDelay {
         float y3 = m_buffer[(iposition + 3) & m_mask];
 
         float delayed_signal = interpolate_cubic(position_frac, y0, y1, y2, y3);
-        delayed_signal = flush_denormals(delayed_signal);
```

Comment:
What was this for and why is it gone in this commit?

Reply:
it's an additional call to `flush_denormals` which turned out to be unnecessary, since there is another `flush_denormals` call several lines down. my apologies, the text i wrote for the PR was not a complete description of the changes.

i don't think this needs separation into two commits since the only purpose is to update from the upstream version, which has a cleaner commit history.

Reply:
Thanks for the explanation!